src/DGtal/geometry/doc/moduleMetrics.dox

```diff
@@ -15,71 +15,60 @@
+- (@e symmetry) @f$d(a,b)=d(b,a)@f$
 - (@e triangular @e inequality) @f$d(a,c) \leq d(a,b) + d(b,c)@f$
 
-When performing some geometry processing, one may consider 
-additional requirements on the metric and thus norms:
+When performing some geometry processing, one may consider  additional requirements on the metric and thus norms:
 
-@definition Given a vector space EV, a @a norm is a map @f$g@f$ from  @f$EV@f$ to a sub-group
-  @f$F@f$ of @f$\mathbb{R}@f$ such that @f$\forall \vec{x},\vec{y}\in EV@f$,
+@definition Given a vector space EV, a @a norm is a map @f$g@f$ from  @f$EV@f$ to a subgroup   @f$F@f$ of @f$\mathbb{R}@f$ such that @f$\forall \vec{x},\vec{y}\in EV@f$,
 -    (@e non-negative) @f$ g(\vec{x})\geq 0@f$
--    (@e identity @e of @e indiscernibles) @f$ g(\vec{x})= 0
-    \Leftrightarrow \vec{x}=\vec{0}@f$
--   (@e triangular @e inequality) @f$g(\vec{x}+\vec{y}) \leq
-    g(\vec{x}) + g(\vec{y})@f$
--    (@e homogeneity) @f$ \forall \lambda\in\mathbb{R}, \quad
-    g(\lambda\cdot\vec{x}) = |\lambda|\cdot g(\vec{x})@f$
+-    (@e identity @e of @e indiscernibles) @f$ g(\vec{x})= 0     \Leftrightarrow \vec{x}=\vec{0}@f$
+-    (@e triangular @e inequality) @f$g(\vec{x}+\vec{y}) \leq     g(\vec{x}) + g(\vec{y})@f$
+-    (@e homogeneity) @f$ \forall \lambda\in\mathbb{R}, \quad     g(\lambda\cdot\vec{x}) = |\lambda|\cdot g(\vec{x})@f$
 
 @f$d(a,b):= g(b-a)@f$ is the metric induced by the norm @f$g@f$.
 
-In the following, we restrict our attention to metric spaces even if
-some of them are induced by norms.
-
-
+In the following, we restrict our attention to metric spaces even if some of them are induced by norms.
 
 @section subConcepts Main concepts
 
 In DGtal, the concept of CMetricSpace is associated with metric space
 definition. A first refinement of this concept is the concept of
 CDigitalMetricSpace whose models implement distance functions on
-integer numbers (i.e. @f$(\mathbb{Z}^n, \mathbb{Z}, d)@f$)). For
+integer numbers (i.e. @f$(\mathbb{Z}^d, \mathbb{Z}, d)@f$)). For
 example,
 
- - @f$ l_p@f$ metrics (ExactPredicateLpSeparableMetric or
- InexactPredicateLpSeparableMetric) are models of CMetricSpace for
- @f$p>1@f$
+- @f$ l_p@f$ metrics (ExactPredicateLpSeparableMetric or  InexactPredicateLpSeparableMetric) are models of CMetricSpace for  @f$p>1@f$
 
- - Chamfer norms with axis symmetric unit ball (see experimental::ChamferNorm2D in dimension 2).
+- Chamfer norms with axis symmetric unit ball (see experimental::ChamferNorm2D in dimension 2).
 
- - DigitalMetricAdapter which adapts any metric (model of
-   CMetricSpace) to digital space using the ceil function. This induces
-   a digital metric (ceil function preserves the
-   triangular inequality).
- 
+- DigitalMetricAdapter which adapts any metric (model of    CMetricSpace) to digital space using the ceil function. This induces    a digital metric (ceil function preserves the    triangular inequality).
```

Comment:
Is that kind of modification voluntary or also due to the editor ? :grin:

Reply:
apparently